src/_posts/security/procedures/2000-01-01-external-testing.md

```diff
@@ -33,7 +33,9 @@ Such service offer is governed by an agreement which engages all three parties i
 This document defines specifically the scope of the pentesting, its duration, and give several restrictions.
 
 {% note %}
-If your service provider does not have three-way agreement, we can provide you with one.
+By default, and to speed up requests, **Scalingo will offer you its audit agreement template**. The document will be systematically transferred to you for review and approval before signature.
+
+In the event that you or your service provider needs to use its own agreement template, it is possible to transfer it to us. It will then be evaluated internally by our competent departments. Additional time may be required to process the request fully.
```

Comment:
```suggestion
If you or your service provider prefer to use your own audit agreement template, you may submit it to us. It will then undergo internal review by the appropriate departments. Please note that this may extend the processing time.
```